backend/src/apiserver/server/pipeline_server.go

```diff
@@ -191,11 +191,14 @@ func (s *PipelineServer) CreatePipelineV1(ctx context.Context, request *apiv1bet
 	}
 
 	// A new pipeline has been created, but pipeline version creation failed after validation
+	// This block should not be executed when an existing pipeline is found
```

Comment:
This code is inside `CreatePipelineV1`. Shouldn't we change this behavior only in the v2 method, and keep v1 behavior as-is--whatever that is in v1.8.* before we refactor?

Reply:
Another question: the issue reported was upload, will `upload` method eventually call into such `create`, or we need to change `upload` method as well?

Reply:
> the issue reported was upload, will upload method eventually call into such create, or we need to change upload method as well?
The upload method directly calls resource manager, and a cleaning logic is added as well.

Reply:
It turned out out that we actually need to change the `CreatePipelineV1` in order to be consistent with the old v1 behavior. I may not be exposed via SDK or UI, but I propose to maintain it the same to stay consistent. The change was probably introduced in the recent refactoring. I changed it back.